wordpress-mobile/WordPress-iOS
# Issue/implement mentions in gutenberg

This PR adds support for adding user mentions in the Block Editor (Gutenberg)

<img src="https://user-images.githubusercontent.com/651601/79641067-1b808200-818d-11ea-974f-100be0f0d82a.gif" width=325>

Gutenberg Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2163

To test:
 - Start the app
 - Create a new post using GB (preferably on a site with multiple users)
 - Add a text block (paragraph, list, heading)
 - Tap on the @ icon on the toolbar
 - Check that the mention interface shows up
 - Tap characters and see if the list is filtered according
 - Select an option
 - Check if mention is added to the content
 - Repeat but this time, clean the suggestion completely or press the X button on the text field
 - Check if no mention is inserted.

PR submission checklist:
